metasploit-framework.gemspec

```diff
@@ -248,4 +248,17 @@ Gem::Specification.new do |spec|
 
   # Needed for multiline REPL support for interactive SQL sessions
   spec.add_runtime_dependency 'reline'
+
+  # Standard libraries: https://www.ruby-lang.org/en/news/2023/12/25/ruby-3-3-0-released/
```

Comment:
> # Standard library updates
> RubyGems and Bundler warn if users do require the following gems without adding them to Gemfile or gemspec. This is because they will become the bundled gems in the future version of Ruby.
> 
> This warning is suppressed if you use bootsnap gem. We recommend to run your application with DISABLE_BOOTSNAP=1 environment variable at least once. This is limitation of this version.
> 
> Targeted libraries are:
> 
> - abbrev
> -  base64
> -  bigdecimal
> -  csv
> -  drb
> -  getoptlong
> -  mutex_m
> -  nkf
> -  observer
> -  racc
> -  resolv-replace
> -  rinda
> -  syslog

This is now an error in Ruby 3.4.0-preview1